core/internal/cltest/factories.go

```diff
@@ -787,3 +787,9 @@ func MustInsertRandomKey(t *testing.T, store *strpkg.Store) models.Key {
 	require.NoError(t, store.CreateKeyIfNotExists(k))
 	return k
 }
+
+func MustInsertOffchainreportingOracleSpec(t *testing.T, store *strpkg.Store) models.OffchainreportingOracleSpec {
+	spec := models.OffchainreportingOracleSpec{}
```

Comment:
Might want a `t.Helper()` here.

Reply:
What does it do?

Reply:
If `require` fails, it'll print the error in your test, rather than on line 793.